src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
> cannot be done, without a bc break I guess.

Yes, it's a BC break. Let's discuss that in a new issue

About this pull request, and as per your suggestion above, we can do the following:
```diff
diff --git a/src/core.js b/src/core.js
index c85669a..a78d457 100644
--- a/src/core.js
+++ b/src/core.js
@@ -451,16 +451,16 @@ $.extend( $.validator, {
 		// http://jqueryvalidation.org/Validator.showErrors/
 		showErrors: function( errors ) {
 			if ( errors ) {
+				var validator = this;
 
 				// Add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
-				for ( var name in errors ) {
-					this.errorList.push( {
-						message: errors[ name ],
-						element: this.findByName( name )[ 0 ]
-					} );
-				}
+				this.errorList = $.map( this.errorMap, function( message, name ) {
+					return {
+						message: message,
+						element: validator.findByName( name ) [ 0 ]
+					};
+				} );
 
 				// Remove items from success list
 				this.successList = $.grep( this.successList, function( element ) {
```